gitHooks/installHooks.py

```diff
@@ -0,0 +1,5 @@
+#!/bin/python
+
+import subprocess
+
+subprocess.run(["git", "config", "--local", "core.hooksPath", "gitHooks/hooks"])
```

Comment:
You don't need python for this :)

Reply:
I assume, you can't :( Create a .bat and .sh file. Otherwise, you require the users to have python installed.